<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>983</width>
    <height>795</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Protocol</string>
  </property>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>701</width>
     <height>681</height>
    </rect>
   </property>
   <attribute name="horizontalHeaderVisible">
    <bool>false</bool>
   </attribute>
   <attribute name="horizontalHeaderStretchLastSection">
    <bool>true</bool>
   </attribute>
   <row>
    <property name="text">
     <string>Project:</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>EUT:</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>Serial-No.:</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>Model-No.:</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>Model-Name:</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>EUT Environment:</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>Company:</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>Group:</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>Technician:</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>Test Procedure:</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>Test Setup:</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>Test Result:</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>TEMPEST Z-No.:</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>Date:</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>Label-No.:</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>Comment:</string>
    </property>
   </row>
   <column>
    <property name="text">
     <string/>
    </property>
   </column>
   <item row="5" column="0">
    <property name="text">
     <string/>
    </property>
   </item>
  </widget>
  <widget class="QGroupBox" name="groupBox_8">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>480</y>
     <width>231</width>
     <height>91</height>
    </rect>
   </property>
   <property name="title">
    <string>Type of Test</string>
   </property>
   <widget class="QWidget" name="verticalLayoutWidget_6">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>214</width>
      <height>51</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_9">
     <item>
      <widget class="QRadioButton" name="RBtnTT1">
       <property name="whatsThis">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Type of Test:&lt;/span&gt;&lt;/p&gt;&lt;p&gt;Zone 1 &amp;amp; 2 (national)&lt;/p&gt;&lt;p&gt;or&lt;/p&gt;&lt;p&gt;Level B &amp;amp; C (internationa)&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="text">
        <string>Zone 1/2 or Level B/C</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="RBtnTT2">
       <property name="whatsThis">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Type of Test&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="text">
        <string>SDIP 27 Level A</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_7">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>310</y>
     <width>231</width>
     <height>171</height>
    </rect>
   </property>
   <property name="title">
    <string>User</string>
   </property>
   <widget class="QWidget" name="verticalLayoutWidget_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>211</width>
      <height>103</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_8">
     <item>
      <widget class="QRadioButton" name="RBtnUser1">
       <property name="whatsThis">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;who uses or get the tested EUT:&lt;/span&gt;&lt;/p&gt;&lt;p&gt;Administration and other organistions (e.g. BMVg)&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="text">
        <string>Administration</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="RBtnUser2">
       <property name="whatsThis">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;who uses or get the tested EUT:&lt;/span&gt;&lt;/p&gt;&lt;p&gt;Military and military administration (e.g. BWI, BAAINBw)&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="text">
        <string>Military</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="RBtnUser3">
       <property name="whatsThis">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;who uses or get the tested EUT:&lt;/span&gt;&lt;/p&gt;&lt;p&gt;Companies&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="text">
        <string>Companies</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="RBtnUser4">
       <property name="whatsThis">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;who uses or get the tested EUT:&lt;/span&gt;&lt;/p&gt;&lt;p&gt;other (e.g. private persons)&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="text">
        <string>other</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_5">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>40</y>
     <width>231</width>
     <height>271</height>
    </rect>
   </property>
   <property name="title">
    <string>Type of EUT</string>
   </property>
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>201</width>
      <height>221</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_5">
     <item>
      <widget class="QRadioButton" name="RBtnTE1">
       <property name="text">
        <string>Laptop, Notebook, Netbook...</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="RBtnTE2">
       <property name="text">
        <string>Tablets, Smartphones ...</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="RBtnTE3">
       <property name="text">
        <string>Server, NAS ...</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="RBtnTE4">
       <property name="toolTip">
        <string>e.g. SINA-Workstation</string>
       </property>
       <property name="text">
        <string>Desktop-PCs ...</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="RBtnTE5">
       <property name="toolTip">
        <string>e.g Elcrodat, SINA-Box</string>
       </property>
       <property name="text">
        <string>Line Decoder ...</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="RBtnTE6">
       <property name="toolTip">
        <string>e.g. Router, Switches</string>
       </property>
       <property name="text">
        <string>active Net Components</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="RBtnTE7">
       <property name="text">
        <string>Crypto Radio</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="RBtnTE8">
       <property name="toolTip">
        <string>e.g Key?, KVZ</string>
       </property>
       <property name="text">
        <string>other ..</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QPushButton" name="BtnGenDocument">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>750</y>
     <width>141</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Generate Document</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BtnOk">
   <property name="geometry">
    <rect>
     <x>880</x>
     <y>750</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Ok</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BtnCancel">
   <property name="geometry">
    <rect>
     <x>780</x>
     <y>750</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbTestNo">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>10</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;TestNo:&lt;/p&gt;&lt;p&gt;[Test]-[Measurement]-[sheet of ]&lt;/p&gt;&lt;p&gt;e.g. M86V0-34-2/6&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="whatsThis">
    <string extracomment="testtet"/>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="text">
    <string>BS-M86V0</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>Report-No.:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BtnAddEnvironment">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>750</y>
     <width>161</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Add EUT Environment</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BtnDelEnvironment">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>750</y>
     <width>161</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Delete EUT Environment</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
